src/services/formatting.js

```diff
@@ -16,3 +16,19 @@ export function toFormattedText(text) {
     </span>
   ));
 }
+
+/*
+ * returns by default
+ * Hello world (font-weight === 600)
+ * for <bold>Hello world<bold>
+ */
+export function toFormattedBoldText(text, fontWeight = 600) {
```

Comment:
This would be much easier and less error prone to just use built-in React Components rather than building custom rendering logic/parsing. You can give React nodes directly in an object (see comment above) and create a `Component` (e.g. `<BoldText>`) directly and not have to parse a string by hand.

Reply:
Got it